packages/theme/webpack.config.js

```diff
@@ -26,62 +29,59 @@ module.exports = () => {
 				},
 				{
 					test: /bootstrap\.scss$/,
-					loader: ExtractTextPlugin.extract({
-						fallback: 'style-loader',
-						use: [
-							{
-								loader: 'css-loader',
-								options: {
-									importLoaders: 3,
-									minimize: true,
-									sourceMap: false,
-								},
+					use: [
+						{
+							loader: isDev ? 'style-loader' : MiniCssExtractPlugin.loader,
+						},
+						{
+							loader: 'css-loader',
+							options: {
+								importLoaders: 3,
+								sourceMap: true,
 							},
-							{
-								loader: 'postcss-loader',
-								options: {
-									plugins: () => [
-										autoprefixer({
-											browsers: ['>0.25%', 'IE 11', 'not op_mini all'],
-										}),
-									],
+						},
+						{
+							loader: 'postcss-loader',
+							options: {
+								postcssOptions: {
+									ident: 'postcss',
+									plugins: [['postcss-preset-env', { browsers: 'last 2 versions' }]],
```

Comment:
Would be nice to have a browserlistrc to set the conf once for all and applied it to all webpack configs including babel and autoprefixer (maybe not part of your PR, just a thought)

Reply:
Sure, even if I think it could be nice to keep this CSS compatible with email for example (not sure that we have to deal with the same compatible in our products)